src/storybook/story-test-runner/extend-stories.ts

```diff
@@ -69,6 +72,17 @@ export function extendStoriesFromStoryFile(
             ) || {};
     }
 
+    for (const story of withStoryFileExtendedStories) {
```

Comment:
If we have remote storybook and it has more stories than our local version, we would still be able to run tests